netlify.toml

```diff
@@ -43,6 +43,20 @@
   status = 202
   force = true
 
+# Supporting external links to a policy which was orignally put on the website with a
```

Comment:
```suggestion
# Supporting external links to a policy which was originally put on the website with a
```

Reply:
The irony of this PR adding more misspellings than it fixes is not lost on me. But I'm not going to worry about misspellings in comments if that's ok.